package/libnmea2000/libnmea2000.mk

```diff
@@ -0,0 +1,22 @@
+################################################################################
+#
+# libnmea2000
+#
+################################################################################
+
+LIBNMEA2000_VERSION = ecd56bcf57f90b860759626b9a248d9e05e7ba50
+LIBNMEA2000_SITE = https://github.com/ttlappalainen/NMEA2000.git
+LIBNMEA2000_SITE_METHOD = git
+LIBNMEA2000_INSTALL_STAGING = YES
+LIBFOO_CONF_OPTS = -DNMEA2000_MSG_DEBUG -DNMEA2000_BUF_DEBUG
```

Comment:
Should this be `LIBNMEA2000` instead of `LIBFOO`?

Reply:
Wow, absolutely.

Didn't end up needing it, so I'll remove it.